readme.md

```diff
@@ -198,7 +199,7 @@ Now, we have only the code we need and can git commit.
 
 * If there is a merge conflict, work together to resolve it!
 
-### Resolving Merge Conflicts `Git Pull (Merge Conflicts)`
+### Resolving Merge Conflicts `git pull (Merge Conflicts)
 
```

Comment:
Check to see how line 202 looks in preview, I see you have an opening '`' but not closing one